src/sequencer/sequencer.h

```diff
@@ -15,7 +15,7 @@ struct Sequencer
         Full = 1,
     };
 
-    enum class Bar : char
+    enum Bar : short
```

Comment:
same here re the `char` vs `short`
also, why not using `enum class` anymore?

Reply:
I wasn't able to find a signed byte type for Teensy 4.1 so short was the smallest type I could find that can hold negative numbers.

The `class` was removed as `enum class` won't compile:

```
In file included from /Users/natehouk/Documents/Arduino/libraries/Midier/src/sequencer/sequencer.cpp:1:
/Users/natehouk/Documents/Arduino/libraries/Midier/src/sequencer/sequencer.h:18:10: warning: elaborated-type-specifier for a scoped enum must not use the 'class' keyword
   18 |     enum class Bar : int8_t
      |     ~~~~ ^~~~~
      |          -----
```